tests/ha/ha_cluster_crash_test.pm

```diff
@@ -45,35 +45,52 @@ sub run {
     # List of things to check
     my @checks = qw(kill-sbd kill-corosync kill-pacemakerd split-brain-iptables);
     my $preflight_start_time = time;
-    my $fencing_start_delay = calculate_sbd_start_delay();
+    #
```

Comment:
Void comment?

Reply:
Oh did I delete the comment somehow ? Just added it back.